vectordatasource/transform.py

```diff
@@ -9471,6 +9470,9 @@ def update_min_zoom(ctx):
         local = defaultdict(lambda: None)
         local.update(props)
         local['zoom'] = zoom
+        # this is to make the name `properties` visible in the queries.yaml's
+        # where clause
+        local['properties'] = props
```

Comment:
This may impact build time, but generally makes things more consistent and obvious.